File: PB - More Exercises Python/2.0 Conditional Statements - More Exercises/07. Flower Shop.py
from math import ceil, floor

count_mgn, count_zum, count_roses, count_cactus, gift_price = [float(input()) for _ in range(5)]
tax = 0.95
total_sum = (count_mgn * 3.25 + count_zum * 4 + count_roses * 3.5 + count_cactus * 8) * tax

if total_sum < gift_price:
    print(f'She will have to borrow {ceil(gift_price - total_sum)} leva.')
else:
    print(f'She is left with {floor(total_sum - gift_price)} leva.')

'''
2
3
5
1
50
'''

File: PB - More Exercises GO/1.0 Simple Operations and Calculations - More Exercises/2.0 Conditional Statements - More Exercises/07. Flower Shop.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var countMgn, countZum, countRoses, countCactus, giftPrice float64
	fmt.Scan(&countMgn, &countZum, &countRoses, &countCactus, &giftPrice)

	tax := 0.95
	totalSum := (countMgn*3.25 + countZum*4 + countRoses*3.5 + countCactus*8) * tax
	if totalSum < giftPrice {
		fmt.Printf("She will have to borrow %v leva.", math.Ceil(giftPrice-totalSum))
	} else {
		fmt.Printf("She is left with %v leva.", math.Floor(totalSum-giftPrice))
	}
}

/*
name   :07. Flower Shop
input  :2 3 5 1 50
output :She will have to borrow 9 leva.
*/
